modules/serverless-functions-podman-macos.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-setup.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-functions-podman-macos_{context}"]
+= Setting up podman on macOS
+
+To use advanced container management features, you might want to use podman with {FunctionsProductName}. To do so on macOS, you need to start the podman machine and configure the Knative (`kn`) CLI to connect to it.
+
+.Procedure
+
+. Create the podman machine:
++
+[source,terminal]
+----
+$ podman machine init --memory=8192 --cpus=2 --disk-size=20
+----
+
+. Start the podman machine, which serves the Docker API on a UNIX socket:
++
+[source,terminal]
+----
+$ podman machine start
+FIXME example output
```

Comment:
Needs fixing?

Reply:
The output is a little big. Will you include it all?

```
Starting machine "podman-machine-default"
Waiting for VM ...
Mounting volume... /Users/myuser:/Users/user

This machine is currently configured in rootless mode. If your containers
require root permissions (e.g. ports < 1024), or if you run into compatibility
issues with non-podman clients, you can switch using the following command: 

	podman machine set --rootful

API forwarding listening on: /Users/myuser/.local/share/containers/podman/machine/podman-machine-default/podman.sock

The system helper service is not installed; the default Docker API socket
address can't be used by podman. If you would like to install it run the
following commands:

	sudo /usr/local/Cellar/podman/4.2.1/bin/podman-mac-helper install
	podman machine stop; podman machine start

You can still connect Docker API clients by setting DOCKER_HOST using the
following command in your terminal session:

	export DOCKER_HOST='unix:///Users/myuser/.local/share/containers/podman/machine/podman-machine-default/podman.sock'

Machine "podman-machine-default" started successfully
```

Reply:
Thanks Jeff, included the relevant parts.